docs/notebooks/topic_coherence_model_selection.ipynb

```diff
@@ -213,24 +154,25 @@
      "output_type": "stream",
      "text": [
       "19998\n",
-      "Dictionary(107980 unique tokens: [u'jbwn', u'porkification', u'sowell', u'sonja', u'luanch']...)\n",
-      "CPU times: user 38.3 s, sys: 2.43 s, total: 40.7 s\n",
-      "Wall time: 43.7 s\n"
+      "Dictionary(24593 unique tokens: [u'woods', u'hanging', u'woody', u'localized', u'gaa']...)\n",
+      "CPU times: user 20 s, sys: 797 ms, total: 20.8 s\n",
+      "Wall time: 21.1 s\n"
      ]
     }
    ],
    "source": [
     "%%time\n",
     "\n",
     "corpus = NewsgroupCorpus(data_path)\n",
```

Comment:
What's a reason to use `NewsgroupCorpus` instead of [sklearn data interface](http://scikit-learn.org/stable/datasets/index.html#the-20-newsgroups-text-dataset)? For comfort?

Reply:
This was a comfort thing -- I already had code for this. I've updated to use the `sklearn.datasets.fetch_20newsgroups` function for retrieving the data.